ext/standard/tests/file/popen_pclose_error-win32.phpt

```diff
@@ -0,0 +1,30 @@
+--TEST--
+Test popen() and pclose function: error conditions on Windows
```

Comment:
This separate Windows test can probably be dropped now?

Reply:
Yep, I completely forgot about this one :(